test/asm/test.sh

```diff
@@ -76,14 +76,19 @@ if ! diff --strip-trailing-cr syntax-error.err $errput; then
 fi
 
 for i in *.asm; do
+	if [ "$i" = "includefile.asm" ]; then
```

Comment:
I considered a general-purpose system of grabbing flags for `$i` from `$i.flags` if it exists, but thought it would be overkill for one test case. (And this test.sh does have hard-coded test names for special cases in a couple of other circumstances.)

Reply:
Other test runners support such flag parsing, so it wouldn't be overkill.

Reply:
...Looks like I didn't commit the `includefile.asm` test anyway. >_< Will fix.